pycbc/events/significance.py

```diff
@@ -197,8 +197,20 @@ def get_n_louder(back_stat, fore_stat, dec_facs,
     Wrapper to find the correct n_louder calculation method using standard
     inputs
     """
+    # This nan to inf conversion is a safety shield; there shouldn't be any
+    # nan statistic values, but if there are, they should be considered as
+    # quiet as possible. nans are considered greater than any floats in an
+    # argsort
+    nanmask = np.isnan(back_stat)
+    if any(nanmask):
+        logging.warning(
+            "Setting %d NaN background statistic values to inf",
+            nanmask.sum(),
+        )
+    back_stat_nonan = np.where(nanmask, -np.inf, back_stat
+
```

Comment:
```suggestion
    # Deal with edge case: we don't expect nan statistic values but if they
    # exist they will ruin the n_louder count, as they are considered larger
    # than floats in an argsort.
    nanmask = np.isnan(back_stat)
    if any(nanmask):
        logging.warning(
            "Setting %d NaN background statistic values to inf",
            nanmask.sum(),
        )
    back_stat_nonan = np.where(nanmask, -np.inf, back_stat)
```
Compactifying the comments, and a bracket seems to have gone missing.